triplea_maps.yaml

```diff
@@ -327,6 +327,33 @@
     <br>Feedback highly welcome, if played.
     <br>Leave feedback at the forums: http://triplea.sourceforge.net/mywiki/Community
     <br>or at the Depot: http://sites.google.com/site/tripleaerniebommel/
+- url: https://github.com/KaiMAD/big_world_1939/blob/master/big_world_1939.zip
```

Comment:
Link is wrong. You can delete your fork at this point most likely. Link should be the latest 'release': ie: "https://github.com/triplea-maps/big_world_1939/releases/download/0.5/big_world_1939.zip"